docs/sources/panels-visualizations/visualizations/state-timeline/index.md

```diff
@@ -19,9 +19,55 @@ weight: 100
 
 # State timeline
 
-State timelines show discrete state changes over time. Each field or series is rendered as its unique horizontal band. State regions can either be rendered with or without values. This visualization works well with string or boolean states but can also be used with time series. When used with time series, the thresholds are used to turn the numerical values into discrete state regions.
+A state timeline visualization displays data in a way that shows state changes over time. In a state timeline, the data is presented as a series of bars or bands called _state regions_. State regions can be rendered with or without values, and the region length indicates the duration or frequency of a state within a given time range.
 
-{{< figure src="/static/img/docs/v8/state_timeline_strings.png" max-width="1025px" caption="state timeline with string states" >}}
+For example, if you're monitoring the CPU usage of a server, you can use a state timeline to visualize the different states, such as “LOW,” “NORMAL,” “HIGH,” or “CRITICAL,” over time. Each state is represented by a different color and the lengths represent the duration of time that the server remained in that state:
+
+{{< figure src="/static/img/docs/state-timeline-panel/state-timeline-panel.png" max-width="1025px" alt="A state timeline panel showing CPU usage" >}}
+
+The state timeline visualization is useful when you need to monitor and analyze changes in states or statuses of various entities over time. You can use one when you need to:
+
+- Monitor the status of a server, application, or service to know when your infrastructure is experiencing issues over time.
+- Identify operational trends over time.
+- Spot any recurring issues with the health of your applications.
+
+## Configure a state timeline
+
+<!-- video TBA here -->
+
+## Supported data formats
+
+The state timeline panel works best if you have data capturing the various states of entities over time, formatted as a table. The data must include:
+
+- **Timestamps** - Indicates when each state change occurred. This could also be the start time for the state change. You can also add an optional timestamp to indicate the end time for the state change.
```

Comment:
```suggestion
- **Timestamps** - Indicate when each state change occurred. This could also be the start time for the state change. You can also add an optional timestamp to indicate the end time for the state change.
```